apps/antidotec_pb/src/antidotec_flag.erl

```diff
@@ -0,0 +1,98 @@
+%% -------------------------------------------------------------------
+%%
+%% Copyright (c) 2014 SyncFree Consortium.  All Rights Reserved.
+%%
+%% This file is provided to you under the Apache License,
+%% Version 2.0 (the "License"); you may not use this file
+%% except in compliance with the License.  You may obtain
+%% a copy of the License at
+%%
+%%   http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing,
+%% software distributed under the License is distributed on an
+%% "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+%% KIND, either express or implied.  See the License for the
+%% specific language governing permissions and limitations
+%% under the License.
+%%
+%% -------------------------------------------------------------------
```

Comment:
I'm a bit confused.. why old copyright with SyncFree for a new file? Did you have to cherry-pick something that have been removed?

Reply:
I added under the assumption that it this applies to the entire repo and is on the files for convenience, but I'm not sure

Reply:
In my view we should not do it, let's check with @thruflo

Reply:
Hmm, good question. How does this work, I wonder? Perhaps it retains its old copyright notice and if we change it, we can add ourselves (Electric DB Limited) as another copyright holder?

Or is it necessary to preserve the old copyright / file headers? I know the Apache license has the "add this to your files, but we don't do it on all our modules and typically the license file in the repo root is where the license is established.

Reply:
I have no clue :)